src/com/lftechnology/java/training/dipak/employeemanagement/dao/DaoFactory.java

```diff
@@ -0,0 +1,31 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.dao;
+
+/**
+ * <p>This class contains method to create objects for the class in the Dao
+ * layer.</p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class DaoFactory {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)